### ISSUE TITLE
Review of benchmark paper

### ISSUE THREAD

**Issue by jbae11** [arfc/2018-bae-benchmark#1]
The benchmark paper is ready for review.

This paper will be prepared for `Annals of Nuclear Energy` appendix submission ( not full paper)

This issue can be closed when the results of the review have been reflected to the paper.

**Comment by katyhuff** [comment]
See the attached. Sorry it's especially scribbly this time. If anything is unclear, please don't hesitate to let me know! 
[2018-bae-benchmark.pdf](https://github.com/arfc/2018-bae-benchmark/files/2253980/2018-bae-benchmark.pdf)

**Comment by jbae11** [comment]
@katyhuff Thank you for taking your time reviewing the paper. I made the edits and pushed to the repo.

For the LWR cycle time, I followed Eva's advice on the cycle time and core mass, since Eva had the same issue with ORION. I learned that the intention with the 4.5 batches and 1 year cycle time in the benchmark was for the continuous-fuel-flow codes to model 1.5 year cycle time, 3 batches reactor.

For changing the Cycamore reactor, I do understand it is a bad move, but without it the results don't quite hold much significance. I do agree that the way Cyclus does it is more correct, and maybe that can be added to the paper.
